pypyr/steps/contextclear.py

```diff
@@ -0,0 +1,46 @@
+"""pypyr step that clears context values.
+
+This is handy if you run the same step multiple times in a pipeline and you
+don't want the previously assigned context settings for that step to impact the
+next iteration.
+"""
+import pypyr.log.logger
+
+# logger means the log level will be set correctly
+logger = pypyr.log.logger.get_logger(__name__)
+
+
```

Comment:
Is the fact that the context is an unknown grab-bag of stuff going to bite us downstream?  Not to be resolved in this PR, but I'd like there to be a declarative way of controlling what ought to be on the context and then validating that the elements are there, and extraneous context is not there.  

Maybe.

Discuss.